src/bitmessageqt/messageview.py

```diff
@@ -3,6 +3,7 @@
 ===============================
```

Comment:
Module docstring should be something like
"""Custom message viewer with support for switching between HTML and plain text rendering, HTML sanitization, lazy rendering (as you scroll down), zoom and URL click warning popup"""

Reply:
Done.